Error starting API

Traceback (most recent call last):
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py", line 25, in <module>
web_1  |     import psycopg2 as Database
web_1  | ModuleNotFoundError: No module named 'psycopg2'
web_1  | 
web_1  | During handling of the above exception, another exception occurred:
web_1  | 
web_1  | Traceback (most recent call last):
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
web_1  |     worker.init_process()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 119, in init_process
web_1  |     self.load_wsgi()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 144, in load_wsgi
web_1  |     self.wsgi = self.app.wsgi()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
web_1  |     self.callable = self.load()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 49, in load
web_1  |     return self.load_wsgiapp()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
web_1  |     return util.import_app(self.app_uri)
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 358, in import_app
web_1  |     mod = importlib.import_module(module)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
web_1  |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
web_1  |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
web_1  |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
web_1  |   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
web_1  |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
web_1  |   File "/app/super_pizzaria/wsgi.py", line 16, in <module>
web_1  |     application = get_wsgi_application()
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
web_1  |     django.setup(set_prefix=False)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
web_1  |     apps.populate(settings.INSTALLED_APPS)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/apps/registry.py", line 114, in populate
web_1  |     app_config.import_models()
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/apps/config.py", line 211, in import_models
web_1  |     self.models_module = import_module(models_module_name)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
web_1  |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
web_1  |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
web_1  |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
web_1  |   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
web_1  |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/models.py", line 2, in <module>
web_1  |     from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/base_user.py", line 48, in <module>
web_1  |     class AbstractBaseUser(models.Model):
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 122, in __new__
web_1  |     new_class.add_to_class('_meta', Options(meta, app_label))
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 326, in add_to_class
web_1  |     value.contribute_to_class(cls, name)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/options.py", line 206, in contribute_to_class
web_1  |     self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/__init__.py", line 28, in __getattr__
web_1  |     return getattr(connections[DEFAULT_DB_ALIAS], item)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/utils.py", line 214, in __getitem__
web_1  |     backend = load_backend(db['ENGINE'])
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/utils.py", line 111, in load_backend
web_1  |     return import_module('%s.base' % backend_name)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py", line 29, in <module>
web_1  |     raise ImproperlyConfigured("Error loading psycopg2 module: %s" % e)
web_1  | django.core.exceptions.ImproperlyConfigured: Error loading psycopg2 module: No module named 'psycopg2'
web_1  | [2020-10-09 22:22:07 +0000] [7] [INFO] Worker exiting (pid: 7)
web_1  | [2020-10-09 22:22:07 +0000] [10] [ERROR] Exception in worker process
web_1  | Traceback (most recent call last):
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py", line 25, in <module>
web_1  |     import psycopg2 as Database
web_1  | ModuleNotFoundError: No module named 'psycopg2'
web_1  | 
web_1  | During handling of the above exception, another exception occurred:
web_1  | 
web_1  | Traceback (most recent call last):
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
web_1  |     worker.init_process()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 119, in init_process
web_1  |     self.load_wsgi()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 144, in load_wsgi
web_1  |     self.wsgi = self.app.wsgi()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
web_1  |     self.callable = self.load()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 49, in load
web_1  |     return self.load_wsgiapp()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
web_1  |     return util.import_app(self.app_uri)
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 358, in import_app
web_1  |     mod = importlib.import_module(module)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
web_1  |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
web_1  |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
web_1  |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
web_1  |   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
web_1  |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
web_1  |   File "/app/super_pizzaria/wsgi.py", line 16, in <module>
web_1  |     application = get_wsgi_application()
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
web_1  |     django.setup(set_prefix=False)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
web_1  |     apps.populate(settings.INSTALLED_APPS)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/apps/registry.py", line 114, in populate
web_1  |     app_config.import_models()
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/apps/config.py", line 211, in import_models
web_1  |     self.models_module = import_module(models_module_name)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
web_1  |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
web_1  |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
web_1  |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
web_1  |   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
web_1  |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/models.py", line 2, in <module>
web_1  |     from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/base_user.py", line 48, in <module>
web_1  |     class AbstractBaseUser(models.Model):
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 122, in __new__
web_1  |     new_class.add_to_class('_meta', Options(meta, app_label))
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 326, in add_to_class
web_1  |     value.contribute_to_class(cls, name)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/options.py", line 206, in contribute_to_class
web_1  |     self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/__init__.py", line 28, in __getattr__
web_1  |     return getattr(connections[DEFAULT_DB_ALIAS], item)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/utils.py", line 214, in __getitem__
web_1  |     backend = load_backend(db['ENGINE'])
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/utils.py", line 111, in load_backend
web_1  |     return import_module('%s.base' % backend_name)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py", line 29, in <module>
web_1  |     raise ImproperlyConfigured("Error loading psycopg2 module: %s" % e)
web_1  | django.core.exceptions.ImproperlyConfigured: Error loading psycopg2 module: No module named 'psycopg2'
web_1  | [2020-10-09 22:22:07 +0000] [10] [INFO] Worker exiting (pid: 10)
web_1  | [2020-10-09 22:22:07 +0000] [8] [ERROR] Exception in worker process
web_1  | Traceback (most recent call last):
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py", line 25, in <module>
web_1  |     import psycopg2 as Database
web_1  | ModuleNotFoundError: No module named 'psycopg2'
web_1  | 
web_1  | During handling of the above exception, another exception occurred:
web_1  | 
web_1  | Traceback (most recent call last):
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
web_1  |     worker.init_process()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 119, in init_process
web_1  |     self.load_wsgi()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 144, in load_wsgi
web_1  |     self.wsgi = self.app.wsgi()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
web_1  |     self.callable = self.load()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 49, in load
web_1  |     return self.load_wsgiapp()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
web_1  |     return util.import_app(self.app_uri)
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 358, in import_app
web_1  |     mod = importlib.import_module(module)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
web_1  |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
web_1  |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
web_1  |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
web_1  |   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
web_1  |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
web_1  |   File "/app/super_pizzaria/wsgi.py", line 16, in <module>
web_1  |     application = get_wsgi_application()
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
web_1  |     django.setup(set_prefix=False)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
web_1  |     apps.populate(settings.INSTALLED_APPS)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/apps/registry.py", line 114, in populate
web_1  |     app_config.import_models()
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/apps/config.py", line 211, in import_models
web_1  |     self.models_module = import_module(models_module_name)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
web_1  |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
web_1  |   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
web_1  |   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
web_1  |   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
web_1  |   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/models.py", line 2, in <module>
web_1  |     from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/base_user.py", line 48, in <module>
web_1  |     class AbstractBaseUser(models.Model):
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 122, in __new__
web_1  |     new_class.add_to_class('_meta', Options(meta, app_label))
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 326, in add_to_class
web_1  |     value.contribute_to_class(cls, name)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/models/options.py", line 206, in contribute_to_class
web_1  |     self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/__init__.py", line 28, in __getattr__
web_1  |     return getattr(connections[DEFAULT_DB_ALIAS], item)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/utils.py", line 214, in __getitem__
web_1  |     backend = load_backend(db['ENGINE'])
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/utils.py", line 111, in load_backend
web_1  |     return import_module('%s.base' % backend_name)
web_1  |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
web_1  |     return _bootstrap._gcd_import(name[level:], package, level)
web_1  |   File "/usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py", line 29, in <module>
web_1  |     raise ImproperlyConfigured("Error loading psycopg2 module: %s" % e)
web_1  | django.core.exceptions.ImproperlyConfigured: Error loading psycopg2 module: No module named 'psycopg2'
web_1  | [2020-10-09 22:22:07 +0000] [8] [INFO] Worker exiting (pid: 8)
web_1  | Traceback (most recent call last):
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 209, in run
web_1  |     self.sleep()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 357, in sleep
web_1  |     ready = select.select([self.PIPE[0]], [], [], 1.0)
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
web_1  |     self.reap_workers()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
web_1  |     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
web_1  | gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
web_1  | 
web_1  | During handling of the above exception, another exception occurred:
web_1  | 
web_1  | Traceback (most recent call last):
web_1  |   File "/usr/local/bin/gunicorn", line 8, in <module>
web_1  |     sys.exit(run())
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in run
web_1  |     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 228, in run
web_1  |     super().run()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 72, in run
web_1  |     Arbiter(self).run()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 229, in run
web_1  |     self.halt(reason=inst.reason, exit_status=inst.exit_status)
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 342, in halt
web_1  |     self.stop()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 393, in stop
web_1  |     time.sleep(0.1)
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
web_1  |     self.reap_workers()
web_1  |   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
web_1  |     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
web_1  | gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
super_pizzaria_web_1 exited with code 1

--- a/cookiecutter.json
+++ b/cookiecutter.json
@@ -6,5 +6,6 @@
     "description": "My awesome pizzaria project",
     "version": "0.1.0",
     "python_version": "3.9.0",
-    "poetry_version": "1.1.0"
+    "poetry_version": "1.1.0",
+    "db_password": "123!@qwe"
 }

--- a/hooks/post_gen_project.py
+++ b/hooks/post_gen_project.py
@@ -1,15 +1,4 @@
-import random
-import string
-
 from shutil import copyfile
-
-
-def create_random_secret_key():
-    chars = string.ascii_letters + string.digits
-    secret_key = "".join(random.choice(chars) for _ in range(32))
-    with open(".secrets.yaml", "w+") as secrets_file:
-        secrets_file.write("default:\n")
-        secrets_file.write(f"  SECRET_KEY: {secret_key}")
 
 
 def creating_env_file():
@@ -21,6 +10,5 @@ def copy_dot_secrets():
 
 
 if __name__ == "__main__":
-    create_random_secret_key()
     creating_env_file()
     copy_dot_secrets()

--- a/{{cookiecutter.module_name}}/.env.template
+++ b/{{cookiecutter.module_name}}/.env.template
@@ -1,6 +1,6 @@
-DJANGO_DEBUG=True
+DJANGO_ENV=localhost
 DJANGO_SETTINGS_MODULE={{cookiecutter.module_name}}.settings
-POSTGRES_PASSWORD=123!@qwe
+POSTGRES_PASSWORD={{cookiecutter.db_password}}
 POSTGRES_USER=postgres
 POSTGRES_DB={{cookiecutter.module_name}}
-DATABASE_URL=postgres://postgres:123!@qwe@db:5432/{{cookiecutter.module_name}}
+DATABASE_URL=postgres://postgres:{{cookiecutter.db_password}}@db:5432/{{cookiecutter.module_name}}

--- a/{{cookiecutter.module_name}}/.secrets.template.yaml
+++ b/{{cookiecutter.module_name}}/.secrets.template.yaml
@@ -1,10 +1,10 @@
 default:
-  SECRET_KEY: GoˆgMhrCcQJmbw2o?AwrQqV53.(0{7!x
+  SECRET_KEY: {{random_ascii_string(32)}}
   DATABASES:
     default:
       ENGINE: django.db.backends.postgresql_psycopg2
-      NAME: super_pizzaria
+      NAME: {{cookiecutter.module_name}}
       USER: postgres
-      PASSWORD: q2Tq9^b_q*WwrHcW
-      HOST: localhost
+      PASSWORD: {{cookiecutter.db_password}}
+      HOST: db
       PORT: 5432

--- a/{{cookiecutter.module_name}}/Dockerfile
+++ b/{{cookiecutter.module_name}}/Dockerfile
@@ -27,6 +27,4 @@ COPY ./pyproject.toml /app/
 
 RUN poetry install $( test "$DJANGO_ENV" = production && echo "--no-dev") --no-interaction --no-ansi
 
-COPY .env_template /app/.env
-
 COPY . /app/

--- a/{{cookiecutter.module_name}}/pyproject.toml
+++ b/{{cookiecutter.module_name}}/pyproject.toml
@@ -5,26 +5,27 @@ description = "{{cookiecutter.description}}"
 authors = ["{{cookiecutter.full_name}} <{{cookiecutter.email}}>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-django = "^3.0.5"
-dynaconf = {version = "^2.2.3", extras = ["yaml"]}
-gunicorn = "^20.0.4"
-djangorestframework = "^3.11.0"
-markdown = "^3.2.2"
-django-filter = "^2.2.0"
+python = "~3.9.0"
+django = "*"
+psycopg2 = "*"
+dynaconf = {version = "*", extras = ["yaml"]}
+gunicorn = "*"
+djangorestframework = "*"
+markdown = "*"
+django-filter = "*"
 djangorestframework-simplejwt = {git = "https://github.com/SimpleJWT/django-rest-framework-simplejwt"}
 
 [tool.poetry.dev-dependencies]
-pytest = "^5.2"
-fake = "^0.8"
-mixer = "^6.1.3"
-pytest-cov = "^2.8.1"
-pytest-django = "^3.9.0"
-pre-commit = "^2.3.0"
-black = "^19.10b0"
-mypy = "^0.770"
-flake8 = "^3.7.9"
-isort = "^4.3.21"
+pytest = "*"
+fake = "*"
+mixer = "*"
+pytest-cov = "*"
+pytest-django = "*"
+pre-commit = "*"
+black = "*"
+mypy = "*"
+flake8 = "*"
+isort = "*"
 
 [build-system]
 requires = ["poetry>=0.12"]

--- a/{{cookiecutter.module_name}}/settings.yaml
+++ b/{{cookiecutter.module_name}}/settings.yaml
@@ -1,12 +1,8 @@
 default:
   # APP specific settings
   module_name: {{cookiecutter.module_name}}
-  server: foo.com
-  username: default user
-  password: false
-  foo: bar
   # Django Required starting settings
-  SECRET_KEY: 1234
+  DEBUG: false
   AUTH_USER_MODEL: user.User
   STATIC_URL: /static/
   STATIC_ROOT: static
@@ -64,10 +60,12 @@ default:
     - NAME: django.contrib.auth.password_validation.NumericPasswordValidator
   # Database
   # https://docs.djangoproject.com/en/2.0/ref/settings/#databases
-  DATABASES:
-    default:
-      ENGINE: django.db.backends.sqlite3
-      NAME: db.sqlite3
+
+localhost:
+  DEBUG: true
+  username: localhost_user
+  foo: bar localhost
+  server: localhostserver.com
 
 development:
   username: dev user
@@ -75,23 +73,21 @@ development:
   server: devserver.com
 
 production:
-  server: prodserver.com
   username: prod user
   foo: bar prod
-  value: this value is for django app
+  server: prodserver.com
 
 staging:
-  server: stagingserver.com
   username: staging user
   foo: bar stag
+  server: stagingserver.com
 
 testing:
-  server: stagingserver.com
+  DEBUG: true
   username: testing user
   foo: bar testing
-
-customenv:
-  server: customserver.com
-
-global:
-  foo: It overrides every other env
+  server: testingserver.com
+  DATABASES:
+    default:
+      ENGINE: django.db.backends.sqlite3
+      NAME: db.sqlite3

